mod_reforged/hooks/ai/tactical/behavior.nut

```diff
@@ -0,0 +1,12 @@
+::mods_hookBaseClass("ai/tactical/behavior", function(o)
```

Comment:
For hooks the convention is to use the bracket on the same line, and space after `function` (basically like adam wrote it in his mod hooks scripts). So: `function (o) {`